addon/components/o-s-s/input-container.hbs

```diff
@@ -31,5 +31,12 @@
   </div>
   {{#if @errorMessage}}
     <span class="text-color-error margin-top-px-6">{{@errorMessage}}</span>
+  {{else if @feedbackMessage}}
+    <span class={{concat "margin-top-px-6 font-color-" this.feedbackMessageType "-500"}}>
+      {{#unless (eq this.feedbackMessageType "error")}}
+        <OSS::Icon @icon={{this.messageIcon}} />
+      {{/unless}}
+      <span>{{@feedbackMessage.value}}</span>
+    </span>
   {{/if}}
```

Comment:
As pointed in [the comment](https://github.com/upfluence/oss-components/pull/453#pullrequestreview-2467504053), the idea was to ALWAYS use the local `this.feedbackMessage` (the snippet literally says `feedbackMessage` and not `feedbackMessageType`)

Now you have a weird situation where given `feedbackMessage` is `{type: "invalid_type", value: "my message"}`, `@feedbackMessage` will be truthy and here is what will be rendered:

```html
<span class="margin-top-px-6 font-color--500">
      <span>my message</span>
    </span>
```

- No type class (as `feedbackMessageType` returned `undefined`): the `font-color--500`
- No icon because `feedbackMessageType` returned `undefined`
- Just a message displayed in black because no valid font color class was applied...

And tests for that case would've pinpointed that